src/ctapipe/calib/camera/calibrator.py

```diff
@@ -227,8 +231,10 @@ def _calibrate_dl1(self, event, tel_id):
         # subtract any remaining pedestal before extraction
         if dl1_calib.pedestal_offset is not None:
             # this copies intentionally, we don't want to modify the dl0 data
-            # waveforms have shape (n_pixel, n_samples), pedestals (n_pixels, )
-            waveforms = waveforms - dl1_calib.pedestal_offset[:, np.newaxis]
```

Comment:
Shouldn't the pedestal have shape `(n_channels, n_pixels)`? We certainly don't expect the pedestal to be the same for different signal paths.